### PR TITLE
LPD-37769 Fixed Publisher Support Contact Info - wrong text on Publisher website URL

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPD-37769](https://liferay.atlassian.net/browse/LPD-37769)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl [hunk 1]
```diff
@@ -86,19 +85,19 @@
...
-			<#assign sanitizedUrl = "https://" + supportUrl />
+			<#assign sanitizedUrl = "https://" + publisherUrl />
 		</#if>
 	</#if>
 
-	<#if supporPhoneFiltered?has_content>
+	<#if supportPhoneFiltered?has_content>
 		<#assign
-			supporPhone = supporPhoneFiltered[0].value
+			supporPhone = supportPhoneFiltered[0].value
```
Comment (welitonpe) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl:100]: Very good, could you fix this error that was already present in the code?

```suggestion
			supportPhone = supportPhoneFiltered[0].value
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl [hunk 2]
```diff
@@ -234,11 +229,9 @@
...
 						<span class="text-black-50">Phone</span>
 
-						<#if supporPhone?has_content>
-							<a class="font-weight-bold" href="tel:${supporPhone}" target="_blank">
-								${supporPhone}
-							</a>
-						</#if>
+						<a class="font-weight-bold" href="tel:${supporPhone}" target="_blank">
+							${supporPhone}
+						</a>
```
Comment (welitonpe) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl:234]: Very good, could you fix this error that was already present in the code?

```suggestion
						<a class="font-weight-bold" href="tel:${supportPhone}" target="_blank">
							${supportPhone}
						</a>
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl (hunk 1 above)
Comment (eduardo-diniz) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/ddm-templates/marketplace-help-and-support/ddm-template.ftl:100]: Yes, of course, I had mapped that out and just forgot to adjust it.
thanks for reminding me

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Thank you @eduardo-diniz

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**